packages/common-types/src/base/Point.ts

```diff
@@ -1,7 +1,7 @@
 import BN from "bn.js";
 
-import { BNString, IPoint, StringifiedType } from "../baseTypes/commonTypes";
-import { ecCurve } from "../utils";
+import { BNString, EllipticCurve, EllipticPoint, IPoint, StringifiedType } from "../baseTypes/commonTypes";
```

Comment:
Could me standardize this Point Class so that it match the mpcCoreKit Point Class?